src/util/string_expr.h

```diff
@@ -114,11 +114,6 @@ inline const array_string_exprt &to_array_string_expr(const exprt &expr)
 class refined_string_exprt : public struct_exprt
 {
 public:
-  DEPRECATED("use refined_string_exprt(length, content, type) instead")
-  refined_string_exprt() : struct_exprt()
-  {
-  }
```

Comment:
This has only been marked as deprecated on 12 Jan 2019.

Reply:
Yes; however, I doubt it ever had any external user at all, given how specific it is to the string refiner.